library/src/blas2/rocblas_ger_strided_batched.cpp

```diff
@@ -0,0 +1,194 @@
+                          batch_count);
+
+            if(layer_mode & rocblas_layer_mode_log_bench)
+                log_bench(handle,
+                          "./rocblas-bench -f ger_strided_batched -r",
+                          rocblas_precision_string<T>,
+                          "-m",
+                          m,
+                          "-n",
+                          n,
+                          "--alpha",
+                          *alpha,
+                          "--incx",
+                          incx,
+                          "--stridex",
+                          stridex,
+                          "--incy",
+                          incy,
+                          "--stridey",
+                          stridey,
+                          "--lda",
+                          lda,
+                          "--strideA",
+                          strideA,
+                          "--batch_count",
+                          batch_count);
+        }
+        else
+        {
+            if(layer_mode & rocblas_layer_mode_log_trace)
+                log_trace(handle,
+                          rocblas_ger_strided_batched_name<T>,
+                          m,
+                          n,
+                          alpha,
+                          x,
+                          incx,
+                          stridex,
+                          y,
+                          incy,
+                          stridey,
+                          A,
+                          lda,
+                          strideA,
+                          batch_count);
+        }
+
+        if(layer_mode & rocblas_layer_mode_log_profile)
+            log_profile(handle,
+                        rocblas_ger_strided_batched_name<T>,
+                        "M",
+                        m,
+                        "N",
+                        n,
+                        "incx",
+                        incx,
+                        "stridex",
+                        stridex,
+                        "incy",
+                        incy,
+                        "stridey",
+                        stridey,
+                        "lda",
+                        lda,
+                        "strideA",
+                        strideA,
+                        "batch_count",
+                        batch_count);
+
+        if(!x || !y || !A)
+            return rocblas_status_invalid_pointer;
+
+        if(m < 0 || n < 0 || !incx || !incy || lda < m || lda < 1 || stridex < m * std::abs(incx)
+           || stridey < n * abs(incy) || strideA < lda * n || batch_count < 0)
+            return rocblas_status_invalid_size;
+
+        // Quick return if possible. Not Argument error
+        if(!m || !n || !batch_count)
+            return rocblas_status_success;
+
+        rocblas_ger_strided_batched_template(
+            handle, m, n, alpha, x, incx, stridex, y, incy, stridey, A, lda, strideA, batch_count);
+
```

Comment:
... batched (and strided_batched for congruency) template_functions should include a "shift" parameter that could be applied to the instances in the batch. If no shift is required, it should be equal zero... in this case:
rocblas_ger_strided_batched_template(
handle, m, n, alpha, x, **0**, incx, stridex, y, **0**, incy, stridey, A, **0**, lda, stride, batch_count);

*see other related comments.